cli/popper/gha.py

```diff
@@ -537,18 +543,21 @@ def __init__(self, action, workspace, env, dry):
         self.cwd = os.getcwd()
 
     def run(self, reuse=False):
-        cmd = self.action.get('runs', ['entrypoint.sh'])
-        cmd[0] = os.path.join('./', cmd[0])
-        cmd.extend(self.action.get('args', []))
-
         cwd = self.cwd
-        if not self.dry_run:
-            if 'repo_dir' in self.action:
-                os.chdir(self.action['repo_dir'])
-                cmd[0] = os.path.join(self.action['repo_dir'], cmd[0])
-            else:
-                os.chdir(os.path.join(cwd, self.action['uses']))
-                cmd[0] = os.path.join(cwd, self.action['uses'], cmd[0])
+        if self.action['uses'] == 'sh':
+            cmd = self.action.get('runs')
```

Comment:
here we also need to consider `args`, e.g.:

```hcl
action "foo" {
  uses = "sh"
  runs = "./mycommand"
  args = ["some", "args"]
}
```

Reply:
In addition, paths are specified with respect to the root of the project, so we need to `cd` to the root of the project as it is done below. More details are available in the [docs](https://popper.readthedocs.io/en/latest/sections/extensions.html#host).